test/common/creation.go

```diff
@@ -47,13 +47,23 @@ func (client *Client) CreateChannelOrFail(name string, channelTypeMeta *metav1.T
 	client.Tracker.Add(gvr.Group, gvr.Version, gvr.Resource, namespace, name)
 }
 
-// CreateChannelsOrFail will create a list of Channel Resources in Eventing.
+// CreateChannelsOrFail will create a list of typed Channel Resources in Eventing or fail the test if there is an error.
 func (client *Client) CreateChannelsOrFail(names []string, channelTypeMeta *metav1.TypeMeta) {
 	for _, name := range names {
 		client.CreateChannelOrFail(name, channelTypeMeta)
 	}
 }
 
+// CreateDefaultChannelOrFail will create a default Channel Resource in Eventing or fail the test if there is an error.
+func (client *Client) CreateDefaultChannelOrFail(channel *messagingv1alpha1.Channel) {
```

Comment:
nit: this creates a generic Channel object with a backing one that is read from the default channel map... It doesn't create a default channel.
Might be clearer if we call it `CreateChannelWithDefaultOrFail`?
But I leave it up to you

Reply:
Yeah that name sounds better.
Done.